TheFoodieNetwork/app/src/main/res/layout/item_friend.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
NICE!!  Seeing semantic changes always makes me happy because it shows that attention-to-detail :-D